samples/Calling/src/app/App.tsx

```diff
@@ -138,6 +159,8 @@ const App = (): JSX.Element => {
           userId={userId}
           displayName={displayName}
           callLocator={callLocator}
+          /* @conditional-compile-remove(PSTN-calls) */
+          alternativeCallerId={alternativeCallerId}
```

Comment:
is it alternateCallerId or alternativeCallerId?
Let's keep it consistent across all PRs and use what's being use in headless sdk